hack/golangci-lint.sh

```diff
@@ -18,11 +18,11 @@ BUILD_TAGS_TUNNEL="$BUILD_TAGS_DEFAULT,remote"
 BUILD_TAGS_REMOTE="remote,containers_image_openpgp"
 
 SKIP_DIRS_ABI=""
-SKIP_DIRS_TUNNEL="pkg/api,pkg/domain/infra/abi,internal/domain/infra/abi"
-SKIP_DIRS_REMOTE="libpod/events,pkg/api,pkg/domain/infra/abi,internal/domain/infra/abi,pkg/machine/qemu,pkg/trust,test"
+SKIP_DIRS_TUNNEL=""
+SKIP_DIRS_REMOTE="libpod/events,pkg/machine/qemu,pkg/machine/wsl,test"
 
 declare -a to_lint
-to_lint=(ABI TUNNEL REMOTE)
```

Comment:
I assume this isn't needed because remote == tunnel?

Reply:
yep, on linux we just lint the same things twice because of that and the linter isn't exactly fast so I dropped it.